src/main/java/com/avides/springboot/springtainer/common/container/EmbeddedContainer.java

```diff
@@ -0,0 +1,5 @@
+package com.avides.springboot.springtainer.common.container;
+
+public interface EmbeddedContainer
```

Comment:
Maybe a comment to avoid an annoying warning

Reply:
Added `// nothing`.